docker/backend/Dockerfile

```
@@ -0,0 +1,34 @@
+# Development Dockerfile for Mobius Backend
+# This Dockerfile is optimized for development with hot reloading
+
+FROM python:3.11-slim
+
+# Set working directory
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)